pySDC/projects/PinTSimE/estimation_check_2capacitors.py

```diff
@@ -59,18 +72,17 @@ def run(dt, use_switch_estimator=True):
 
     # fill description dictionary for easy step instantiation
     description = dict()
-    description['problem_class'] = battery_2condensators  # pass problem class
+    description['problem_class'] = battery_n_capacitors  # pass problem class
     description['problem_params'] = problem_params  # pass problem parameters
     description['sweeper_class'] = imex_1st_order  # pass sweeper
     description['sweeper_params'] = sweeper_params  # pass sweeper parameters
     description['level_params'] = level_params  # pass level parameters
     description['step_params'] = step_params
```

Comment:
Instead of generating the description object from scratch, you could separate the `run` function into `generate_description` and `run`. Then you can put the `generate_description` function into `batter_2capacitors_model.py` and in the run here you only change the parameters you need different. Meaning
```
def run(dt):
    description = generate_description()
    description['level_params']['dt'] = dt
    controller = ...
    controller.run(...)
    ...
```
The same applies to the estimation check for a single capacitor.

Reply:
I did another kind of doing this. I haven't pushed it yet, because I want to change some other things that you mentioned. I hope you will be satisfied with the new changes.